--- configure.in	2016-09-02 18:34:21.273781660 +0000
+++ configure.in	2016-09-02 18:34:33.185631908 +0000
@@ -15,6 +15,7 @@
 # Checks for programs.
 AC_PROG_CC
 AC_PROG_LD
+m4_ifdef([AM_PROG_AR], [AM_PROG_AR])
 AC_PROG_INSTALL
 AC_PROG_AWK
 AC_PROG_CPP
@@ -36,8 +37,8 @@
 
 AC_EXEEXT
 
-dnl more automake stuff
-AM_C_PROTOTYPES
+#dnl more automake stuff
+#AM_C_PROTOTYPES
 
 dnl libtool
 AC_DISABLE_STATIC	
