--- lib/stdio.in.h	2013-09-26 19:51:34.000000000 +0000
+++ lib/stdio.in.h	2016-09-02 13:32:38.557363626 +0000
@@ -119,7 +119,9 @@
    so any use of gets warrants an unconditional warning.  Assume it is
    always declared, since it is required by C89.  */
 #undef gets
+#if defined(__GLIBC__) && !defined(__UCLIBC__) && !__GLIBC_PREREQ(2, 16)
 _GL_WARN_ON_USE (gets, "gets is a security hole - use fgets instead");
+#endif
 
 #if @GNULIB_FOPEN@
 # if @REPLACE_FOPEN@
